.github/workflows/builds.yml

```diff
@@ -31,7 +31,56 @@ jobs:
             type=semver,pattern={{version}}
             type=semver,pattern={{major}}.{{minor}}
             type=semver,pattern={{major}}
-
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: rudderlabs
```

Comment:
Not sure if I commented about this already but I think it makes sense to have `rudderlabs` as a secret as well. Sorry if you already answered I must've missed it.

Reply:
The problem with having this as a secret is that it puts ***** on several places on our logs. For instance image names, it is very easy to figure the secret by that and makes things inconvenient.